go.mod

```diff
@@ -69,3 +69,5 @@ require (
 	k8s.io/api v0.21.3
 	k8s.io/apimachinery v0.21.3
 )
+
+replace github.com/containers/common => github.com/vrothberg/common v0.0.3-0.20210720091711-39e7214a081e
```

Comment:
Looks like this is still a WIP?

Reply:
Thank you for catching that!  I missed to remove that line.